test/verify/check-service

```diff
@@ -39,46 +36,21 @@ class TestService(composerlib.ComposerCase):
         b.wait_text(".pf-c-empty-state__content .pf-c-title", "No blueprints")
 
         # create blueprint with random name and description
-        name_template = string.ascii_letters + string.digits + "-" + "_"
-        bp_name = "".join(random.sample(name_template, 20))
-        desc_template = string.ascii_letters + string.digits + " "
-        bp_desc = "".join(random.sample(desc_template, 40))
+        bp_name = "bp_name"
+        bp_desc = "bp_description"
         b.click(".pf-c-empty-state button:contains('Create blueprint')")
-        b.wait_visible("div[role=dialog] #cmpsr-modal-crt-blueprint")
+        b.wait_visible("#cmpsr-modal-crt-blueprint")
         # blueprint name
         b.set_input_text("#textInput-modal-markup", bp_name)
         # blueprint description
         b.set_input_text("#textInput2-modal-markup", bp_desc)
         # create
         b.click("#create-blueprint-modal-create-button")
-        b.wait_not_present("div[role=dialog] #cmpsr-modal-crt-blueprint")
+        b.wait_not_present("#cmpsr-modal-crt-blueprint")
 
-        # wait for all available compontents visible
+        # wait for back to blueprint link and click
         with b.wait_timeout(300):
```

Comment:
Can you remove this timeout. The standard timeout should be good enough for blueprint creation so if it is taking 5 minutes we should fail.

Reply:
Done!